src/cpp/session/SessionModuleContext.cpp

```diff
@@ -2390,7 +2390,24 @@ bool isSessionTempPath(FilePath filePath)
 
 std::string sessionTempDirUrl(const std::string& sessionTempPath)
 {
-   if (session::options().programMode() == kSessionProgramModeDesktop)
+   // In desktop mode we can have R itself handle requests for content in
+   // the session temporary folder.
+   bool useRHelpServer = session::options().programMode() == kSessionProgramModeDesktop;
```

Comment:
nit: would it be worth making this a `static` and initialize it from a helper function? (since the value won't ever change in a given session)